UI-session-7/javascript.js

```diff
@@ -0,0 +1,356 @@
+function onBlurForm2() {
+
+    switch (currentField) {
+
+        case "vehName":
+            var vehicleName = document.getElementById("vname").value;
+            if (vehicleName == "") {
+                document.getElementById("error").innerHTML = "required";
+                return;
+            }
+            document.getElementById("error").innerHTML = " ";
+            document.getElementById("vehName").style.display = "none";
+            label.innerHTML = " whats the type of " + vehicleName;
+            document.getElementById("vehType").style.display = "block";
+            currentField = "vehType";
+
+            break;
+
+        case "vehType":
+            var vehicleType = document.getElementById("vtype").value;
+            var vehicleName = document.getElementById("vname").value;
+
+            if (vehicleType == "") {
+                document.getElementById("error").innerHTML = "required";
+                return;
+            }
+            document.getElementById("error").innerHTML = " ";
+
+            document.getElementById("vehType").style.display = "none";
+            label.innerHTML = "Mention the number of " + vehicleName;
+            document.getElementById("vehNumber").style.display = "block";
+            currentField = "vehNumber";
+
+            break;
+
+        case "vehNumber":
+            var vehNumber = document.getElementById("vnumber").value;
+            var vehicleName = document.getElementById("vname").value;
+
+            if (vehNumber == "") {
+                document.getElementById("error").innerHTML = "required";
+                return;
+            }
+            document.getElementById("error").innerHTML = " ";
+
+            document.getElementById("vehNumber").style.display = "none";
+            label.innerHTML = "Tell something about " + vehicleName;
+            document.getElementById("vehDescription").style.display = "block";
+            document.getElementById("next2").style.display = "block";
+            currentField = "vehDescription";
+
+            break;
+
+        default:
+    }
+
+
+}
+
+
+/* switch case for plan and pricing  form for switching input fields */
+function onBlurForm3() {
+
+    switch (currentField) {
+
+        case "currency":
+            var currency = document.getElementById("currency").value;
+            var vehicleType = document.getElementById("vtype").value;
+
+            document.getElementById("currType").style.display = "none";
+            label.innerHTML = "choose paln for your " + vehicleType;
+            debugger;
+            if(document.getElementById('daily').checked){
+                moneypass = document.getElementById("daily").innerText+" "+document.getElementById("currency").value;
+            }
+            else if(document.getElementById('monthly').checked){
+                moneypass = document.getElementById("monthly").innerText+" "+document.getElementById("currency").value;
```

Comment:
Try to avoid global variables by simplifying logic like below:
```
	let moneypass = '';
    document.getElementById("p_Registration").style.display ="none";
	document.getElementsByName('passtype').forEach(function(item){if(item.checked==true){moneypass= item.value;}});
	moneypass = moneypass +" "+document.getElementById("currency").value;
    label.innerHTML = "Thank you for registering " +document.getElementById("ename").value +" Kindly Pay ->"+moneypass;
```

Reply:
used "let" keyword to change scope of variable from global to functional